.github/workflows/build_and_test_master.yml

```diff
@@ -146,6 +146,8 @@ jobs:
      Test-e2e_react-component-library:
       runs-on: ubuntu-latest
       needs: [Build_icon_library, Lint_react-component-library]
+      matrix:
```

Comment:
FYI it's not building at the moment – `matrix` should be nested under a `strategy` key

Reply:
🤦‍♂️